sql2.txt

```diff
@@ -2,44 +2,44 @@
 to join the OrderDetails table with the product table.  For TotalCount, you will Use
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
-
+SELECT products.ProductName, SUM(OrderDetails.Quantity) AS TotalCount  FROM OrderDetails JOIN products ON OrderDetails.ProductID = products.ProductID GROUP BY OrderDetails.ProductID ORDER BY ProductName;
 
 2. Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
 ordered by customer name. Paste your SQL statement below. You will have to join the Customers
-table with the Orders table, and the Orders table with the OrderDetails table, 
+table with the Orders table, and the Orders table wOith the OrderDetails table, 
 because the Orders table is the one that ties the Customer table to the OrderDetails table.
-
+SELECT Customers.customerName,  OrderDetails.OrderDetailID FROM Customers JOIN  OrderDetails ON Customers.CustomerID = Orders.CustomerID JOIN Orders ON Orders.OrderID = OrderDetails.OrderID ORDER BY Customers.customerName;
```

Comment:
kindly modify your code the questions says "Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
ordered by customer name. Paste your SQL statement below. You will have to join the Customers
table with the Orders table, and the Orders table with the OrderDetails table, 
table with the Orders table, and the Orders table with the OrderDetails table, 
because the Orders table is the one that ties the Customer table to the OrderDetails table. "

this code `JOIN  OrderDetails ON Customers.CustomerID `should be JOIN "`Orders`" not "`OrderDetails`" and this code JOIN Orders ON Orders.OrderID = OrderDetails.OrderID  should be JOIN "`OrderDetails`" not "`Orders`"